lib/knapsack_pro/report.rb

```diff
@@ -5,7 +5,7 @@ def self.save
 
       if test_files.empty?
         KnapsackPro.logger.warn("No test files were executed on this CI node.")
-        KnapsackPro.logger.debug("When you use knapsack_pro regular mode then probably reason might be very narrowed tests list - you run only tests with specified tag and there are fewer test files with the tag than node total number.")
+        KnapsackPro.logger.debug("When you use knapsack_pro Regular Mode, the reason for no tests executing might be a very narrow tests list. Most likely, you run only tests with a specified tag, and the number of test files with the tag was lower than the number of parallel CI nodes.")
```

Comment:
Nice improvements. :+1: 

the last bit could be simplified to ", and there were fewer test files with the tag than parallel CI nodes."